dashboards/apiserver.libsonnet

```diff
@@ -272,6 +272,11 @@ local singlestat = grafana.singlestat;
           sort=1,
         )
       )
+      .addRow(
+        row.new(
+          title='Notice: The SLO (service level objective) and other metrics displayed on this dashboard are for informational purposes only and are not related to any contractual SLA.',
```

Comment:
I think we don't know that the numbers here imply SLO only, for some users it will imply contractual SLA for some don't.

I think this message should live outside this repository.

Reply:
> for some users it will imply contractual SLA for some don't.

We thought others vendors/users might benefit that these are meant as informational only rather than contractual, but thats a good point! Happy to move this into openshift only @metalmatze

Reply:
Yes, definitely good to ask the discussion here. But if it's not generic enough we'll simply put it downstream :)

Reply:
I'm fine if we could figure out more generic wording for this.

Reply:
More than happy to, any suggestions @povilasv?

Reply:
Would the following work:
> Notice: The SLO (service level objective) and other metrics displayed on this dashboard are for informational purposes only.

Reply:
That sounds better :+1:

Reply:
I think the other important part is, that what is represented here is what SIG-Scalability uses to test Kubernetes releases, no more.

Reply:
Or actually it's even softer, as this is only "modeled" after what SIG Scalability uses, they don't even use exactly this (although it might actually be useful to them, so we might want to have that conversation with them regardless :) ).